testUtils/prismaSetup.js

```diff
@@ -0,0 +1,8 @@
+import { execSync } from 'child_process'
+
+export default () => {
+  // Clear previous test database and migrations used by Prisma
+  execSync('rm -rf ./testUtils/prisma/migrations && rm -rf ./testUtils/prisma/dev.db')
+  // Create a new database to be used by the Prisma tests
+  execSync('npx prisma migrate dev --name init --schema ./testUtils/prisma/schema.prisma')
```

Comment:
just to make sure:`npx` is something that is always installed with the runtime? is it additional requirement? Can you maybe do it in plain node (instead of `import { execSync } from 'child_process'`).
<not mandatory>

Reply:
`npx` comes bundled with `npm` since npm@5, so I don't a reason why it won't work (we use npm@6, which I need to bump up actually - as the repo is working with Node 14 that Prisma does not support. Adam got us covered in his will [PR](https://github.com/lumigo-io/utils/pull/501/) 😂 )